docs/guides/platform/disk-images/copying-a-disk-image-over-ssh/index.md

```diff
@@ -26,11 +26,13 @@ If the amount of data on your disk is much less than the size of the disk, then
 
 ## Download a Disk over SSH
 
-### Boot into Rescue Mode
+This guide was written to support Downloading a disk over SSH from a  **Linode**, which will be used to Create and Send an image to a separate receiving system that you personally own. This means that the guide will have you switching between the execution of commands on both the **Origin Linode** as the origin of your disk image, and the **Receiving System** which will receive a copy of this image. You will see the terms Origin Linode and Receiving System used to describe these two separate environments.
 
-1.  Prepare the receiving computer by verifying that SSH is installed. Most Linux/Unix-like systems include OpenSSH in their package base by default. If the receiving system is Microsoft Windows, there are multiple SSH solutions available such as [Cygwin and PuTTY](/docs/networking/ssh/using-ssh-on-windows).
+While this guide has been written to accommodate computers running Linux as their operating system, if the receiving system is Microsoft Windows, there are multiple SSH solutions available such as [Cygwin and PuTTY](/docs/networking/ssh/using-ssh-on-windows) which can alternatively be used to complete this process.
```

Comment:
missing the word running: "if the receiving system is running Microsoft Windows,"